src/lib/dialog/dialog-config.ts

```diff
@@ -69,5 +69,14 @@ export class MdDialogConfig {
   /** ID of the element that describes the dialog.  */
   ariaDescribedBy?: string | null = null;
 
+  /** Aria label to assign to the dialog element */
+  ariaLabel?: string | null = null;
+
+  /**
+   * Selector for an element to be focused when the dialog is opened. If omitted or the
+   * element is not found, the dialog will focus the first focusable element.
+   */
+  focusOnOpen?: string | null = null;
```

Comment:
With this API, it's not immediately clear what selector you'd set here to focus the root dialog element.

Reply:
I'm not a big fan of magic-string APIs. I'd rather go with `boolean | string | null`, where the boolean means the host element. 

What do you think of making this PR just add the `ariaLabel` and the `focusOnOpen` for the root element while leaving the content focusing as a responsibility of the focus trap?

Reply:
Sounds good.